src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,16 @@
 import React from 'react';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => (
+  <ul className="movies">
+    {movies.map((movie) => (
+      <MovieCard movie={movie} />
+    ))}
+  </ul>
```

Comment:
- If you use `ul` the next child shouild be `li` tag. You have `div`
- Always add `key` attribute when rendering lists.